appmonitoring/ts/src/package.json

```diff
@@ -22,6 +22,8 @@
     "http-proxy-agent": "7.0.0",
     "https-proxy-agent": "7.0.0",
     "log4js": "^6.9.1",
+    "node-forge": "^1.3.1",
```

Comment:
Let's not have different versions in dev and normal dependencies, should only be normal, no dev dependencies